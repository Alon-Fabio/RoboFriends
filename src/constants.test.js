import renderer from 'react-test-renderer';
import {shallow} from 'enzyme';
const constants = require('./constants')

// let wraper
// beforeEach(()=>{
//     wraper = renderer.create(constants).toJSON()
// })
describe("Constants", ()=>{
    it("Snapshot", ()=>{
        expect(constants).toMatchSnapshot()
    })
})