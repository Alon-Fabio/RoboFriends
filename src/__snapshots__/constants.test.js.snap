// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Constants Snapshot 1`] = `
Object {
  "CHANGE_SEARCH_FIELD": "CHANGE_SEARCH_FIELD",
  "REQUES_ROBOTS_FAILED": "REQUES_ROBOTS_FAILED",
  "REQUES_ROBOTS_PENDING": "REQUES_ROBOTS_PENDING",
  "REQUES_ROBOTS_SUCCESS": "REQUES_ROBOTS_SUCCESS",
}
`;
